composer.json

```diff
@@ -15,11 +15,17 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.5",
-        "bamarni/composer-bin-plugin": "^1.4"
+        "bamarni/composer-bin-plugin": "^1.4",
+        "symfony/process": "^5.2"
```

Comment:
Surprised this isn't included by default.It's a great way to deal with running processes.